lib/WebService/Stripe.pm

```diff
@@ -70,6 +70,10 @@ method get_balance(:$headers) {
     return $self->get( "/v1/balance", {}, headers => $headers );
 }
 
+method get_balance_history(:$headers) {
```

Comment:
hm that does seem a bit weird though, doesn't it? I mean its `/balance/history` on the stripe side... should we try to match their nomenclature?